Projects/Checkers/Data/KnowledgeBase.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace Checkers.Data;
+
+/// <summary>
+/// Stores the starting position and any custom permutation for testing
+/// </summary>
+public static class KnowledgeBase
+{
+	public static List<Piece> GetStartingPosition()
+	{
+		var retVal = new List<Piece>
+			{
+				new() { NotationPosition ="A3", Side = PieceColour.Black},
+				new() { NotationPosition ="A1", Side = PieceColour.Black},
+				new() { NotationPosition ="B2", Side = PieceColour.Black},
+				new() { NotationPosition ="C3", Side = PieceColour.Black},
+				new() { NotationPosition ="C1", Side = PieceColour.Black},
+				new() { NotationPosition ="D2", Side = PieceColour.Black},
+				new() { NotationPosition ="E3", Side = PieceColour.Black},
+				new() { NotationPosition ="E1", Side = PieceColour.Black},
+				new() { NotationPosition ="F2", Side = PieceColour.Black},
+				new() { NotationPosition ="G3", Side = PieceColour.Black},
+				new() { NotationPosition ="G1", Side = PieceColour.Black},
+				new() { NotationPosition ="H2", Side = PieceColour.Black},
+
+				new() { NotationPosition ="A7", Side = PieceColour.White},
+				new() { NotationPosition ="B8", Side = PieceColour.White},
+				new() { NotationPosition ="B6", Side = PieceColour.White},
+				new() { NotationPosition ="C7", Side = PieceColour.White},
+				new() { NotationPosition ="D8", Side = PieceColour.White},
+				new() { NotationPosition ="D6", Side = PieceColour.White},
+				new() { NotationPosition ="E7", Side = PieceColour.White},
+				new() { NotationPosition ="F8", Side = PieceColour.White},
+				new() { NotationPosition ="F6", Side = PieceColour.White},
+				new() { NotationPosition ="G7", Side = PieceColour.White},
+				new() { NotationPosition ="H8", Side = PieceColour.White},
+				new() { NotationPosition ="H6", Side = PieceColour.White}
+			};
+
+		return retVal;
+	}
+
+	public static List<Piece> GetLimitedStartingPosition() => new()
+		{
+			new() { NotationPosition = "H2", Side = PieceColour.Black},
+			new() { NotationPosition = "A1", Side = PieceColour.Black},
+			new() { NotationPosition = "G3", Side = PieceColour.White},
+			new() { NotationPosition = "E5", Side = PieceColour.White}
+		};
```

Comment:
@Wycott The "LimitedStartingPosition" was only for testing purposes right? If yes, then I want to remove it from the game code, but I would be happy to add a unit testing project to the repo. We just didnt have unit tests for any of the other games yet so I haven't added a unit testing project yet.

Reply:
Hi. Yes, I just used that for testing specific cases really. On the subject of feature flags, there is also code to log the game moves to the temp directory which we might want to remove too.

I do have a unit test project for this but wondered if it might be overkill. I think I remember seeing a PR from someone else about adding tests. Wasn't sure if it was a moot point still. Happy to check it in if you think it might be useful.

Reply:
Thanks. I removed the logging and the `LimitedStartingPosition` for now. Some players may like to have a trace log of their game, so I could see that being added back in, but I decided to remove it for now.